app/csr/error.tsx

```diff
@@ -0,0 +1,24 @@
+"use client"
+import { Button } from "@/components/ui/button";
+import { useRouter } from "next/navigation";
+
+export default function Error({
+	error,
+	reset,
+}: {
+  error: Error & { digest?: string }
+  reset: () => void
+})
+{
+	const router = useRouter();
```

Comment:
I was recently updating ESLint and Prettier on my current project, and it might have affected a bit. I added a basic Prettier configuration to avoid such issues.